src/renderer/viz/expressions/base.js

```diff
@@ -66,7 +66,10 @@ export default class Base {
      *
      */
     eval (feature) {
-        throw new CartoRuntimeError('Unimplemented');
+        if (this.isFeatureDependent()) {
```

Comment:
This is being fixed and tracked here https://github.com/CartoDB/carto-vl/pull/1046, I would remove this change here in order to track it there, what do you think?

Reply:
It's cherry-picked from that same commit.

After david's revision, I'm gonna merge that one to master and update this branch, so there should be no problem